samples/wifi/shell/boards/nrf54h20dk_nrf54h20_cpuapp_soc1.overlay

```diff
@@ -3,6 +3,7 @@
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
+#include <common/nordic_nrf_next/nrf54h20_ram0x.dtsi>
```

Comment:
please, review all these files are they do not seem to be updated for upstream usage

Reply:
Sure, thanks. I cherry picked this `next`, will check and align to upstream.